packages/elements/src/containers/Provider.tsx

```diff
@@ -1,15 +1,9 @@
 import * as React from 'react';
 
 import { MockUrlResult } from '../components/TryIt/mocking-utils';
-import { IActiveInfo, NodeIconMapping } from '../types';
+import { IActiveInfo } from '../types';
 
-export interface IProvider extends IActiveInfo {
-  authToken?: string;
-  isStoplightProjectComponent?: boolean;
-}
-
-const defaultIcons: NodeIconMapping = {};
-export const IconsContext = createNamedContext<NodeIconMapping>('IconsContext', defaultIcons);
+export interface IProvider extends IActiveInfo {}
```

Comment:
```suggestion
```

No need to duplicate - `IProvider` doesn't actually extend `IActiveInfo` here

Reply:
After removing it, you can use `IActiveInfo` in `Provider` instead of `IProvider`

Reply:
I am leaving this for now.

It might prove to be useful soon.

If not, I will be voting for removing Provider entirely, since it only wraps 1 other provider and does nothing else.